src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/ImportTranscriptCard.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { FormattedMessage, injectIntl } from '@edx/frontend-platform/i18n';
+import {
+  ActionRow,
+  Button,
+  Icon,
+  IconButton,
+  Stack,
+} from '@edx/paragon';
+import { Close } from '@edx/paragon/icons';
+
+import messages from './messages';
+import { thunkActions } from '../../../../../../data/redux';
+
+export const ImportTranscriptCard = ({
+  setOpen,
+  // redux
+  importTranscript,
+}) => (
+  <Stack gap={3} className="border rounded border-primary-200 p-4">
+    <ActionRow className="h5">
+      <FormattedMessage {...messages.importHeader} />
+      <ActionRow.Spacer />
+      <IconButton
+        src={Close}
+        iconAs={Icon}
+        onClick={() => setOpen(false)}
```

Comment:
I think you can write a test for this similar to the behavior inspection in the EditorContainer test file.